Source/WebKit/WebProcess/WebPage/WebPage.h

```diff
@@ -695,20 +709,20 @@ class WebPage : public API::ObjectImpl<API::Object::Type::BundlePage>, public IP
     void removePluginView(PluginView*);
 #endif
 
-    inline bool isVisible() const;
-    inline bool isVisibleOrOccluded() const;
+    bool isVisible() const;
+    bool isVisibleOrOccluded() const;
 
-    OptionSet<WebCore::ActivityState> activityState() const { return m_activityState; }
+    OptionSet<WebCore::ActivityState> activityState() const;
     bool isThrottleable() const;
 
     LayerHostingMode layerHostingMode() const { return m_layerHostingMode; }
     void setLayerHostingMode(LayerHostingMode);
 
 #if PLATFORM(COCOA)
     void updatePluginsActiveAndFocusedState();
```

Comment:
@darinadler - I was looking into different stuff but noticed that this is unused and don't have any reference. So I noticed that you are doing this work, so rather than doing another PR, I thought to mention so you can get rid of it as well.

Reply:
OK, I’ll remove it.

Reply:
What about in `WebProcess::initializeWebProcess` where there is code that writes it out?

Reply:
Maybe I don’t know what you mean by "this". Are you referring to `layerHostingMode` rather than `activityState`?

Reply:
I think removing it should be a separate patch; I was looking into `layerHostingMode` and there is code that sends messages across processes to set it.

Reply:
Sorry - I am not super familiar with GitHub review tool. I am referring to this: `void updatePluginsActiveAndFocusedState();`.

Reply:
<img width="685" alt="image" src="https://github.com/WebKit/WebKit/assets/52317531/4906714b-66ca-4f3b-855c-2dce036acc01">
It  has only this reference here and not used anywhere else in WebKit code.

Reply:
Yes, I’ll remove `updatePluginsActiveAndFocusedState`.